lib/ahoy/intercom/tracker.rb

```diff
@@ -0,0 +1,26 @@
+require 'intercom'
+require 'ahoy/intercom/client'
+
+module Ahoy
+  module Intercom
+    module Tracker
+      include Client
+
+      def track(event_name, properties = {})
```

Comment:
If it was a store, then the method would be named 'track_event'. And since this is only a tracker that saves events in the intercom, it named it that. But I think the more right name 'save_to_intercom'